.travis.yml

```diff
@@ -126,7 +127,6 @@ before_install:
 install:
     - mkdir -p ~/.config/matplotlib
     - echo "backend:agg" > ~/.config/matplotlib/matplotlibrc
-    - ./scripts/travis/install-openblas.sh $OPENBLAS_ROOT
```

Comment:
Read the commit message which added `install-openblas.sh`: https://github.com/NVIDIA/DIGITS/commit/e6be9fef83eb18419c836c116c9a41956928b610

If you stop building openblas locally, then you'll need to address this issue with Torch. Either don't cache the torch build, force Torch to stop building openblas from source, or find a way to re-install the openblas build which was cached along with the Torch build.